maps_dashboards/ DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,74 @@
+## Developer Guide
+
+So you want to contribute code to this project? Excellent! We're glad you're here. Here's what you need to do.
+
+- [Forking](#forking)
+- [Install Prerequisites](#install-prerequisites)
+- [Setup](#setup)
+- [Run](#run)
+- [Test](#test)
+- [Submitting Changes](#submitting-changes)
+- [Backports](#backports)
+
+### Forking
+
+Fork the repository on GitHub.
+
+### Install Prerequisites
+
+You will need to install [node.js](https://nodejs.org/en/), [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md), and [yarn](https://yarnpkg.com/) in your environment to properly pull down dependencies to build and bootstrap the plugin.
+
+
+### Setup
+
+1. Download the OpenSearch Dashboards source code for the version specified in package.json you want to set up.
+2. Change your node version by `nvm use <version>`to the version specified in `.node-version` inside the OpenSearch Dashboards root directory.
+3. Create a `plugins` directory inside the OpenSearch Dashboards source code directory, if `plugins` directory doesn't exist.
+4. cd into `plugins` directory in the OpenSearch Dashboards source code directory.
+5. Check out this package from version control into the `plugins` directory.
+```
+git init
+// Change to your forked repo link in below command
+git remote add -f origin https://github.com/opensearch-project/dashboards-maps.git
```

Comment:
I think we need set up for both plugins. Shall we add something like
to checkout maps:
... 
...
...

to check out custom import
...
...
..

Reply:
For development, we don't need to check out all the two plugins. The sparse-checkout will help only check out `/maps-dashboards` folder, so we only need to set up maps-dashboards plugin.